simcase-game/src/components/Header.js

```diff
@@ -97,78 +99,94 @@ function Header() {
   function closeComponent() {
     setIsComponentOpen(false);
     setModal(null);
+    closeDialog();
   }
 
   return (
     <header css={headerStyle}>
-      <h2
-        css={{
-          width: "33%",
-          fontSize: "1.1em"
-        }}
-      />
-      <a href="/" css={titleLogo}>
-        <img
-          src="/assets/img/wharton_logo.png"
-          alt="Wharton Logo"
-          css={{ height: 60 }}
-        />
-        <h1 css={{ display: "none" }}>Line Balancing</h1>
-      </a>
-      <div>
+      <div css={titleDivs}>
+        <h2>&nbsp;</h2>
+      </div>
+      <div css={{ ...titleDivs }}>
+        <a href="/" css={titleLogo}>
+          <img
+            src="assets/img/wharton_logo.png"
```

Comment:
I think `src="assets/img/wharton_logo.png"` this won't help you because In some cases it won't show the image for that reason I would recommend you to add the "/" behind assets like this : 
`src="/assets/img/wharton_logo.png"`

Reply:
seems it's working fine with the correction. 10x